.github/workflows/workflow.yml

```diff
@@ -51,7 +51,7 @@ jobs:
 
       - name: Commit openapi.yaml and otg.proto
         id: commit_generated
-        if: github.event_name == 'pull_request'
+        # if: github.event_name == 'pull_request'
```

Comment:
This was done intentionally - otherwise, artifacts were not getting generated on code submission. Ashutosh will verify with snappi team if this new option should be removed permanently.